### ISSUE TITLE
Remove Disconnect references from Shavar services

### ISSUE THREAD

**Issue by skim1102** [mozilla-services/shavar-prod-lists#111]
# About this Issue
After merging the changes from https://github.com/mozilla-services/shavar-prod-lists/pull/110 I saw that the staging S3 showed that there were files updated when we expected to updates:
<img width="1365" alt="Screen Shot 2020-02-03 at 1 23 57 PM" src="https://user-images.githubusercontent.com/25109943/73687784-5e53b300-4690-11ea-8b51-040f7ba4d82e.png">
Diffs of the logs before and after merging #110 shows that the order of the domains have changed causing the list to be "changed" (look at the diff between the .log files attached). 
[ads-track-digest256-BEFORE.log](https://github.com/mozilla-services/shavar-prod-lists/files/4150091/ads-track-digest256-BEFORE.log)
[ads-track-digest256.log](https://github.com/mozilla-services/shavar-prod-lists/files/4150092/ads-track-digest256.log)
We should wait to apply #110 to versioned branches so that the domains are added alphabetically to prevent not-real update like this one. We should also remove all the Disconnect category references in the services and configs.

# Acceptance Criteria
- [ ] We should wait to apply #110 to versioned branches so that we deploy meaningful updates and not a simple re-ordering of the items in the list
- [ ] The domains added in the list should be alphabetical to prevent not-real update like this one
- [ ] We should also remove all the Disconnect category references in the services and configs

**Comment by boolean5** [comment]
Hi @skim1102, in continuation of https://github.com/mozilla-services/shavar-list-creation/pull/121, I was thinking about sending a PR that removes references to the Disconnect category and to `disconnect_mapping.json` from the shavar-prod-lists repository. This would address the 3rd A/C of this issue. Should I go ahead or have you started working on this already?

**Comment by skim1102** [comment]
@boolean5 I went ahead and added the issue from shavar-list-creation repo in the description of the issue. Let's wait to get all the acceptance criteria fulfilled before we check the third acceptance criteria. Alternatively, the second criteria:

>The domains added in the list should be alphabetical to prevent not-real update like this one

has not been worked on yet and would be a good extension for the work you have been doing to remove Disconnect category and JSON file references. Would you like to pick it up as an extended work you have been doing?

**Comment by boolean5** [comment]
Yes, working on the 2nd A/C sounds good. Thanks for suggesting this. I'll get back to you if I have any questions.

**Comment by boolean5** [comment]
@skim1102, some more context on this would be very helpful.

It is not clear to me how the application of https://github.com/mozilla-services/shavar-prod-lists/pull/110 caused the order of domains to change. Could you please explain this a little? 

Was there a domain that was added out of alphabetical order in https://github.com/mozilla-services/shavar-prod-lists/pull/110? Does it have to do with the way the [Disconnect list parser](https://github.com/mozilla/trackingprotection-tools/blob/master/trackingprotection_tools/DisconnectParser.py) parses `disconnect-blacklist.json`?

**Comment by skim1102** [comment]
@boolean5 certainly! I apologize for the delayed response.
> It is not clear to me how the application of #110 caused the order of domains to change. Could you please explain this a little?

Yes, as you have mentioned, the Disconnect list parser uses `.add` as seen [here](https://github.com/mozilla/trackingprotection-tools/blob/master/trackingprotection_tools/DisconnectParser.py#L208) which does not impose a specific order (since `Set` is an unordered collection). However, when the remapping of the Disconnect category domains into the respective categories happened [here](https://github.com/mozilla-services/shavar-prod-lists/pull/110), you can see that the companies and their domains were added in alphanumerical order. This resulted in the list being "changed" even though the contents/domains in the list were not changed. While we can't enforce the ordering in trackingprotection-tools, we can do so in this repo and prevent a list from being "updated" due to changes in the order of the domains written to the file/list.

**Comment by skim1102** [comment]
Also, if you intend to keep working on this issue, please assign yourself to https://github.com/mozilla-services/shavar-list-creation/issues/132. Thank you!

**Comment by boolean5** [comment]
Thanks for explaining this!

I can't self-assign https://github.com/mozilla-services/shavar-list-creation/issues/132 because I don't have write permissions for that repository. I still intend to work on it though, so feel free to assign it to me.
